tests/test_max_stable_set.py

```diff
@@ -1522,3 +1523,22 @@ def test_combining_two_changesets_with_overlap(assert_semantically_equal, partia
             {"routes": {"service_1_route_2"}, "services": {"bus_service"}},
         ),
     ]
+
+
+@pytest.mark.parametrize(
+    "case_name,model_output,expected",
+    [
+        (
+            "simple stops",
+            ["x['stop_3.link:link_7_8_car']", "x['stop_1.link:link_4_5_car']"],
+            ["stop_3.link:link_7_8_car", "stop_1.link:link_4_5_car"],
+        ),
+        (
+            "stops with x in name",
+            ["x['x_stop.link:1234']", "x['stopx.link:1234']", "x['stxop.link:1234']"],
+            ["x_stop.link:1234", "stopx.link:1234", "stxop.link:1234"],
+        ),
+    ],
+)
+def test_processing_model_results(case_name, model_output, expected):
```

Comment:
Great to see a dedicated unit test for this 👍 

As with the function under test though, I am not sure about the name.  How would you describe what we expect this new function to do? The current test name doesn't give me any clue about the behaviour we're verifying here. Is there a way to work the expected behaviour into the test name in a `test_does_something_when_something` style?